packages/sdk-ts/src/client/indexer/types/explorer-rest.ts

```diff
@@ -225,3 +225,19 @@ export interface CW20BalanceExplorerApiResponse {
 }
 
 export { BaseTransaction }
+
+export interface BankTransferFromExplorerApiResponse {
+  sender: string
+  recipient: string
+  amounts: { denom: string; amount: string }[]
```

Comment:
let's use `import { Coin } from '@injectivelabs/ts-types'` to replace the `{ denom: string; amount: string }`